src/api/getRandomArticleTitle.js

```diff
@@ -0,0 +1,19 @@
+
+export const getRandomArticleTitle = lang => {
+  const baseUrl = `https://${lang}.wikipedia.org/w/api.php`
+  const params = {
+    action: 'query',
+    format: 'json',
+    formatversion: 2,
+    origin: '*',
+    list: 'random',
+    rnnamespace: 0,
+    rnlimit: 1
+  }
+  const url = baseUrl + '?' + Object.keys(params).map(p => `${p}=${params[p]}`).join('&')
```

Comment:
want to try [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/toString)?

```js
const params = new URLSearchParams({
    action: 'query',
    format: 'json',
    formatversion: 2,
    origin: '*',
    list: 'random',
    rnnamespace: 0,
    rnlimit: 1
})

console.log(params.toString());

// action=query&format=json&formatversion=2&origin=*&list=random&rnnamespace=0&rnlimit=1
```

Reply:
Nice! I didn't know that.

I'll make a second PR to refactor this and search to use it and maybe encapsulate the MW API some more.